checks/fortisandbox_disk

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+# author: Oguzhan Cicek, OpenSource Security GmbH - oguzhan(at)os-s.de
+
+# Example Input:
+#[...]
+#.1.3.6.1.4.1.12356.118.3.1.1.0 v2.52-build0340 (GA)
+#.1.3.6.1.4.1.12356.118.3.1.2.0 2
+#.1.3.6.1.4.1.12356.118.3.1.3.0 4
+#.1.3.6.1.4.1.12356.118.3.1.4.0 260459760
+#.1.3.6.1.4.1.12356.118.3.1.5.0 710698
+#.1.3.6.1.4.1.12356.118.3.1.6.0 3754417
+#.1.3.6.1.4.1.12356.118.3.1.7.0 2
+#.1.3.6.1.4.1.12356.118.3.1.8.0 32908349
+#.1.3.6.1.4.1.12356.118.3.2.1.0 5.2.50534
+#.1.3.6.1.4.1.12356.118.3.2.2.0 2.4.20034
+#.1.3.6.1.4.1.12356.118.3.2.3.0 3.2.279
+#.1.3.6.1.4.1.12356.118.3.2.4.0 4.478
+#.1.3.6.1.4.1.12356.118.3.2.5.0 14.613
+#[...]
+
+# Example GUI Output:
+# OK	FortiSandbox Disk usage	5.28% used (694.04 GB of 3.58 TB)
+
+factory_settings["fortisandbox_disk_default_levels"] = {
+    "disk_usage": (80, 90),
+}
+
+
+def inventory_fortisandbox_disk(info):
+    for _ in info:
+        yield (None, "fortisandbox_disk_default_levels")
+
+
+def convert_size(size_bytes):
+    if size_bytes == 0:
+        return "0 MB"
+    i = 0
+    size_name = ("MB", "GB", "TB", "PB", "EB", "ZB", "YB")
+    while size_bytes >= 1000:
+        size_bytes = float(size_bytes / 1024)
+        i = i + 1
+    return "%.2f %s" % (size_bytes, size_name[i])
+
+
+def check_fortisandbox_disk(item, params, info):
+    warn, crit = params["disk_usage"]
+    dusage = float(info[0][0][0])
+    dcap = float(info[0][0][1])
+    dperc = float(dcap / dusage)
```

Comment:
Hi Ralf,
I started to look into this PR and I think this line is wrong. To get the usage, you want to compute
`dusage / dcap * 100`,
i.e., used disk space or all available disk space, right? The factor of 100 to get a percentage is also missing.
Best
Jörg